sample-applications/README.md

```diff
@@ -10,11 +11,21 @@ aws ssm list-command-invocations --command-id $containersBuildCommandId
 ```
 
 ### Deploy Applications
+
+*If you are using Kubernetes master v1.14, then use the following command*
+
 ```powershell
 ./Deploy-Applications.ps1 -Namespace $gMSAnamespace -ServiceAccount $serviceaccount -CredSpecResourceName "$credspecResourceName" -SQLSAPassword "$sqlSAPassword" -GMSAUser "$adDirectoryShortName\$gMSAAccountName`$" 
 ```
 
+*If you are using Kubernetes master v1.16 and beyond, then use the following command*
+
+```powershell
+./Deploy-Applications.ps1 -Namespace $gMSAnamespace -ServiceAccount $serviceaccount -CredSpecResourceName "$credspecResourceName" -SQLSAPassword "$sqlSAPassword" -GMSAUser "$adDirectoryShortName\$gMSAAccountName`$" -MVCSampleApp "containermvc-web_v16.yaml" -BookstoreDBApp "containersql-bookstore_v16.yaml" 
```

Comment:
nit - should we rename 'containermvc-web_v16.yaml' and 'containersql-bookstore_v16.yaml'? since it applies to v1.16 and beyond, it could be more generic. 
Maybe, you could rename the other two files used specifically for v1.14.